ID53 - Cleaned up form, added sensitive data guide

itarpeh opened the pull request:
-Bootstrap formatting 
-Link to Umich sensitive data guide

itarpeh committed:
ID53 - Cleaned up form, added sensitive data guide

gsilver commented:
LGTM!

A few notes I left in the code review should be addressed.

A few details that we can address after the merge:

1. Reset state when changing choices. For example - if I choose **App**, then select options including DB options, then change my mind and choose **Simple Website**, the DB options block is still displayed
2. Adding back your styles.

Also - it would be good to not check in things that are just by products, like .DS_Store, etc. To do so:

From terminal:

1. cd ~
2. touch .gitignore
3. brackets .gitignore
4. add, one per line

.DS_Store
node_modules

this way they will not show up as files to add and commit and you can keep using `git add .`

gsilver commented on src/views/order.html:
```diff
@@ -1,66 +1,182 @@
+                URL : {{orderModel.url}}
+              </div>-->
+              <label for="url-input">Choose a URL</label>
+              <input type="url" ng-model=orderModel.url class="form-control" placeholder="Enter url">
```
**orderModel.url** needs to be quoted. Also type="url" is legit, but they are only going to be entering a fragment, so we are not going to be validating as a **url** and Angular seems to void (goes from null to nada when you start typing) so we are better of with type="text"

gsilver commented on src/views/order.html:
```diff
@@ -1,66 +1,182 @@
+            
+            <div id="shortcode" style="margin-top:50px;">
+              <label for="shortcode-input">Enter a shortcode</label>
+              <input type="text" ng-model=orderModel.shortcode class="form-control" placeholder="Enter shortcode">
```
**orderModel.shortcode** needs to be quoted.

gsilver commented on src/views/order.html:
```diff
@@ -1,66 +1,182 @@
-    <div id="url">
-      <div ng-show="orderModel.url !==null">
-        URL : {{orderModel.url}}
+      <!-- this sets the app choices if set -->
```
Let's remove commented out code through out too.

itarpeh commented on src/views/order.html:
```diff
@@ -1,66 +1,182 @@
     </div>
-  </div>
-  </div
+</div>
```
test comment

itarpeh committed:
Update order.html

-Removed commented out code
-Change url input type to "text"
-Fixed ng-model quote issues
